aw/Helpers/AWApiHelper.swift

```diff
@@ -104,6 +104,10 @@ struct AWApiHelper {
         reach.gageId = Int32(newReach.gageId ?? 0)
         reach.gageMetric = Int16(newReach.gageMetric ?? 0)
 
+        if let updatedSecondsString = newReach.lastGageUpdated, let updatedSecondsAgo = Int(updatedSecondsString) {
```

Comment:
Can you split these statements? It's a little much to read all at once. Also might be worth adding a comment as to what's going on cause we won't remember in six months.